package.json

```diff
@@ -60,7 +60,7 @@
     "start": "npm-run-all -p start:css start:js",
     "start:msw": "REACT_APP_MOCK_API=msw npm run start",
     "build": "npm-run-all build:css build:js",
-    "start:js": "react-scripts start",
+    "start:js": "react-scripts --openssl-legacy-provider start",
```

Comment:
Why this flag is required?

Reply:
The client tests won't run on my computer without the flag. If it causes problems, feel free to remove it.

Reply:
We can change it in case it create problems.